app/assets/javascripts/discourse/app/components/choose-topic.js

```diff
@@ -39,6 +39,18 @@ export default Component.extend({
     }
   },
 
+  didInsertElement() {
+    this._super(...arguments);
+
+    schedule("afterRender", () => {
+      $("#choose-topic-title").keydown((e) => {
```

Comment:
If you're binding an event like this you should have an equivalent tear down to remove it.

Reply:
Okay, this is now done.